gui/create-item.lua

```diff
@@ -255,15 +258,24 @@ function createCorpsePiece(creator, bodypart, partlayer, creatureID, casteID, ge
   item_type = "GLOB"
  elseif CORPSE_PIECES[layerName] or HAIR_PIECES[layerName] then -- check if hair
   item_type = "CORPSEPIECE"
+ elseif LIQUID_PIECES[layerName] then -- check if hair
+  item_type = "LIQUID_MISC" -- prolly need to do more work on this (it doesn't like spawning LIQUID items), but it works for now
+  liquid = true
  end
  local itemType = dfhack.items.findType(item_type..":NONE")
  local itemSubtype = dfhack.items.findSubtype(item_type..":NONE")
  local material = "CREATURE_MAT:"..raceName..":"..layerMat
  local materialInfo = dfhack.matinfo.find(material)
  local item_id = dfhack.items.createItem(itemType, itemSubtype, materialInfo['type'], materialInfo.index, creator)
  local item = df.item.find(item_id)
- item:setQuality(quality)
- local isCorpsePiece = false
+ if liquid then
+  local bucket = df.item.find(dfhack.items.createItem(18, -1, 22, 191, creator))
```

Comment:
I'm not sure what you intended `22, 191` to mean, but in my test world it resolves to "sliver barb seed" which doesn't seem to make much sense as a bucket material.

Reply:
this is 100% intentional. 🐇

Reply:
i figured the material of the bucket didn't matter as it was not the item being specified by the user, so, 🐇 

also might help to differentiate those buckets from intentionally spawned buckets.